### PR TITLE
issue-687

### PR DESCRIPTION
**Pull request by mnyakushev**
solving issue #687

### CODE REVIEW

File: template/fr.opensagres.xdocreport.template.velocity/src/main/java/fr/opensagres/xdocreport/template/velocity/cache/XDocReportEntryResourceLoader.java [hunk 1]
```diff
@@ -76,7 +77,7 @@ public Reader getResourceReader(String source, String encoding) throws ResourceN
             InputStream inputStream = cacheInfo.getInputStream();
             if ( inputStream != null )
             {
-                return new InputStreamReader(inputStream);
+                return new InputStreamReader(inputStream, Charset.forName(encoding));
```
Comment (angelozerr) [inline template/fr.opensagres.xdocreport.template.velocity/src/main/java/fr/opensagres/xdocreport/template/velocity/cache/XDocReportEntryResourceLoader.java:80]: What about if encoding is null?

Comment (mnyakushev) [reply template/fr.opensagres.xdocreport.template.velocity/src/main/java/fr/opensagres/xdocreport/template/velocity/cache/XDocReportEntryResourceLoader.java]: You are right. I didn't think that the encoding could be null.

Comment (angelozerr) [reply template/fr.opensagres.xdocreport.template.velocity/src/main/java/fr/opensagres/xdocreport/template/velocity/cache/XDocReportEntryResourceLoader.java]: i dont master the velocity code but I think you should take care of encoding which is null or empty.

Comment (mnyakushev) [reply template/fr.opensagres.xdocreport.template.velocity/src/main/java/fr/opensagres/xdocreport/template/velocity/cache/XDocReportEntryResourceLoader.java]: Yes,  I watched velocity code. It's really possible that encoding will be null.
I fixed this and created another pull request.

**Pull request closed without merging**